src/types/patch.ts

```diff
@@ -11,6 +11,10 @@ export interface ModuleCodeChanges {
   rawLink: string;
   fileDiffs: FileDiff[];
 }
+export interface Parameter {
+  key: string;
+  value: string;
+}
```

Comment:
We dont need to declare this type again we can just pull the type from the generated types. See below comment

Reply:
Do i also not need to declare Parameter in the Patch definition in this file?